profilegen/src/res/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Microsoft Corporation
```

Comment:
Just noticed this out of curiosity: Should we really license to year `2015` ?

Reply:
Changed it to 2017

Reply:
I'd say check what we have in other places on this, license may or may not be the same for 2017